webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/usermanagement/index/MappingIndex.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.schema.descriptors.usermanagement.index;
+
+import io.camunda.webapps.schema.descriptors.usermanagement.UserManagementIndexDescriptor;
+
+public class MappingIndex extends UserManagementIndexDescriptor {
+  public static final String INDEX_NAME = "mappings";
+  public static final String INDEX_VERSION = "8.7.0";
+
+  public static final String MAPPING_KEY = "mappingKey";
+  public static final String CLAIM_NAME = "claimName";
+  public static final String CLAIM_VALUE = "claimValue";
```

Comment:
💭 I think these can be removed, I used them in other handlers to follow a pattern but I'm not sure, beyond historical exporter reasons, if they are used for anything without our context here. Maybe @Zelldon could clarify?